pkg/orchestrator/evaluation/inmemory_broker.go

```diff
@@ -815,3 +816,10 @@ func (b *InMemoryBroker) registerMetrics() (metric.Registration, error) {
 	}, orchestrator.EvalBrokerReady, orchestrator.EvalBrokerInflight, orchestrator.EvalBrokerPending,
 		orchestrator.EvalBrokerWaiting, orchestrator.EvalBrokerCancelable)
 }
+
+func (b *InMemoryBroker) HandleNodeEvent(ctx context.Context, info models.NodeInfo, evt manager.NodeEvent) {
```

Comment:
For readability add a compile time assertion to the top of this file:
`var _ manager.NodeEventHandler = &InMemoryBroker{}`

Reply:
Done in ff7828f3f